docs/cloud/cloud-databases/cloud-db-versions.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cloud database versions
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Database version information
+{: .no_toc }
+
+{% include /cloud-db/cloud-summary-db-version.md %}
+
+{% include page-toc.md %}
+
+## Before you begin
+* [Learn about Cloud databases](/docs/cloud/cloud-databases/cloud-db-manage)
+* [Create a FeatureBase database](/docs/cloud/cloud-databases/cloud-db-create)
+
+{: .note}
+A database that fails to respond will display their version as "Unknown".\
```

Comment:
This is probably on me -- "their" isn't good. 

The issue is an "unknown" database version.

So the note would be:

A database with an unknown version has failed to respond to queries.